lib/Jsrt/JsrtHelper.cpp

```diff
@@ -188,7 +188,11 @@ void JsrtCallbackState::ObjectBeforeCallectCallbackWrapper(JsObjectBeforeCollect
 
 #ifndef _WIN32
         // put something into key to make sure destructor is going to be called
-        pthread_setspecific(s_threadLocalDummy, malloc(1));
+        int res = pthread_setspecific(s_threadLocalDummy, malloc(1));
+        if (res != 0)
+        {
+            abort();
```

Comment:
This can be too much for production. Do we have to abort or do anything? We don’t actually have much control here.